src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +56,8 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    public String getPrintableString() {
```

Comment:
Better to have the @Override keyword to explicitly specify to the reader that this is an overridden method. Moreover, if you make a common mistake of misspelling a method name or not correctly matching the parameters, you will be warned.